### PR TITLE
Add author guide and reviewer guide

### LINKED ISSUE #1: Finish porting the other pages into the website
So that we can leave google docs behind :)

### PR DESCRIPTION
**Pull request by mjskay**
This closes #1 by:

- Adds the author guide and reviewer guide
- Changes the "Submit" nav element to "Submissions" and puts "How to submit", "Author guidelines", and " Reviewer guidelines" as sub-elements under "Submissions". I felt this was more discoverable than a sidebar, which is the other option, but happy to discuss. See [here](https://quarto.org/docs/websites/website-navigation.html#hybrid-navigation) for more on the different possibilities.
  - Note: I put the OJS submission instructions that were in the author guide in submit.qmd instead since I think they belong there better.
- Tweaks the style of deeply-nested headings a bit to make them more legible
- Removes the old `styles.css` which is superceded by `theme.scss`

### CODE REVIEW

**Comment by mjskay** [comment]
For reference, if folks don't want to have to render this PR just to see how the nav changes, it now looks like this:

image

And then if you click on Submissions:

image

File: submit.qmd [hunk 1]
```diff
@@ -17,6 +18,28 @@ You may prepare your paper with any academic paper template; we recommend the [A
...
+3. Upload submission:
+    * Click *Add file*, upload main PDF
+    * Select "article text" as type
+    * Upload other files, if desired. For example, if you are submitting anonymously, you may need to upload non-anonymous versions of some information for the editors to review. See the [author guide](author-guide.qmd#sections).
+    * Click *Save and continue*
+4. Enter Metadata:
+    * Add title and abstract
+    * Add co-authors: click *add contributor*, fill form
+        * Select *Author* as role
+        * *Include contributor in browse lists* -> people can be found from internal search
```
[the remark this answers is not in the document]
Comment (floe) [reply submit.qmd:38]: Ah, if you create accounts for your co-authors and tick that box ("Include contributor in browse lists"), then the co-author accounts can later be found in search boxes.

**Review by floe — APPROVED** [review]
Looks good to me!

File: submit.qmd [hunk 2]
```diff
@@ -35,7 +35,7 @@ To submit:
     * Add title and abstract
     * Add co-authors: click *add contributor*, fill form
         * Select *Author* as role
-        * *Include contributor in browse lists* -> people can be found from internal search
+        * If you create accounts for your co-authors, we recommend ticking *Include contributor in browse lists* so that co-authors' accounts can be found later in search boxes.
```
Comment (mjskay) [inline submit.qmd:38]: Thanks @floe for the clarification. Does this new text sound right?

**Review by floe — APPROVED** [review]
👍

**Pull request merged**